BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/ContentType/ContentTypeFormView.js

```diff
@@ -94,18 +95,18 @@ class ContentTypeFormView extends mix(AbstractFormView).with(FormViewButtonsMixi
             contentTypeId: contentTypeId,
             name: name
         });
-        let message = new FlashMessage(data, 'success');
-        FlashMessageBag.addMessageFlash(message);
         Backbone.Events.trigger('form:deactivate', this);
         Backbone.history.navigate(url, true);
     }
 
     /**
-     * Delete
-     * @param {event} event
+     * Delete content type
      */
-    _deleteElement(event) {
-        let contentType = new ContentType({'content_type_id': this._contentTypeId});
+    _deleteElement() {
+        if (null === this._contentTypeId) {
```

Comment:
Is this test really useful as delete button appear only in edit context so with this._contentTypeId not null?